lib/spack/docs/analyze.rst

```diff
@@ -9,30 +9,43 @@
 Analyze
 =======
 
-The analyze commands allows you to take an existing package install, and run
-an analyzer to extract one or more metrics. The metrics can range from reading
-metadata about install files and output to extracting Application Binary
-Interface (ABI) information from binaries. 
+
+The analyze command is a front-end to various tools that let us analyze
+package installations. Each analyzer is a module for a different kind
+of analysis that can be done on a package installation, including (but not
+limited to) binary, log, or text analysis. Thus, the analyze command group
+allows you to take an existing package install, choose an analyzer,
+and extract some output for the package using it.
+
 
 -----------------
 Analyzer Metadata
 -----------------
 
-For all analyzers, we write to an ``analyze`` folder in the package install
-directory metadata folder. For example, here we see an analyze folder that is
-alongside a wget install:
+For all analyzers, we write to an ``analyzers`` folder in your spack ``$HOME``
```

Comment:
That's fair, but probably also slightly unfortunate.  This can be set with an env var can't it? I recall the XDG issue didn't go anywhere, but is it always actually at that location?

Reply:
I can change to that. I was thinking verbatim "A spack hidden directory in $HOME == spack home directory," haha.

Reply:
This is fixed.

Reply:
@trws The misc cache location can be configured (not with an env var) but things outside it actually can’t.